interface/backend/cases/models.py

```diff
@@ -39,3 +51,5 @@ class Nodule(models.Model):
     candidate = models.OneToOneField(Candidate, on_delete=models.CASCADE, null=True)
 
     centroid = models.OneToOneField('images.ImageLocation', on_delete=models.CASCADE)
+
+    lung_orientation = models.CharField(max_length=1, choices=LungOrientations.choices(), default='0')
```

Comment:
``CharField`` smells wrong to me given that we can want to enforce numbers.

In addition the default should prbobably be ``LungOrientations.none`` rather than "0", if you see the semantic difference.